### ISSUE TITLE
COPY failed:  . . allure-docker-api: no such file or directory

### ISSUE THREAD

**Issue by bbookman** [fescobar/allure-docker-service#154]
Apologies from a docker newbie if I "did the wrong thing" wishing to push an allure instance to GCP.

Using https://github.com/fescobar/allure-docker-service/blob/master/docker/Dockerfile.bionic
 
`gcloud builds submit  . . .`
`Step 19/67 : COPY allure-docker-api $ROOT_DIR/allure-docker-api
COPY failed: stat /var/lib/docker/tmp/docker-builder133308070/allure-docker-api: no such file or directory`

What did I do wrong?  Are there steps someplace to help a Docker newbie get success pushing your Docker container to gcp?  (I have the container running locally and can even get reports)

**Comment by brucebookman** [comment]
Good point, it came from Docker HUB.  Are you suggesting I try that instead?

**Comment by fescobar** [comment]
@bbookman if you want to create your own image you need to clone this full repository and run the command from the root project. That error is because you don't have the `allure-docker-api` directory in your machine. This one https://github.com/fescobar/allure-docker-service/tree/master/allure-docker-api but as I said before you need to clone the complete repo.

The same, if you want to use the allure-docker-service you don't need to build your own image. You can pull directly from the docker hub. You can do that by executing the next commands.

https://github.com/fescobar/allure-docker-service#single-project---local-reports

https://github.com/fescobar/allure-docker-service#multiple-projects---remote-reports

I recommend you to learn docker from scratch. There are a lot of excellent courses in Udemy.
